src/providers/42.js

```diff
@@ -0,0 +1,20 @@
+export default function FortyTwo(options) {
+  return {
+    id: '42',
```

Comment:
```suggestion
    id: '42-school',
```

In addition, could you also rename this file to `42-school.js` please?